src/components/ComboBox/MultiComboBox.tsx

```diff
@@ -183,19 +183,20 @@ export const MultiComboBox: FC<Props> = ({
         <List themes={theme}>
           {selectedItems.map(({ value, label, deletable = true }, i) => (
             <li key={i}>
-              <SelectedItem themes={theme}>
+              <SelectedItem themes={theme} disabled={disabled}>
                 <SelectedItemLabel themes={theme}>{label}</SelectedItemLabel>
 
                 {deletable && (
                   <DeleteButton
                     type="button"
                     themes={theme}
                     className={DELETE_BUTTON_CLASS_NAME}
+                    disabled={disabled}
                     onClick={() => onDelete({ value, label })}
                   >
                     <FaTimesCircleIcon
                       size={11}
-                      color={theme.palette.TEXT_BLACK}
+                      color={disabled ? 'inherit' : theme.color.TEXT_BLACK}
```

Comment:
説明不足ですいません。
`disabled` ではないときもボタンカラーがあるのでどちらも `inherit` でいけます。分岐が不要です。

```suggestion
                      color="inherit"
```

Reply:
@uknmr :aa: 気づかなくてごめんなさい
修正 push しました :pray: